src/components/dapp-staking-v2/my-staking/MyRewards.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="wrapper--cards">
+    <div class="card">
+      <p>
+        Total Staked
```

Comment:
and here

Reply:
@impelcrypto I think all issues seems resolved as your feedback. please check again.